frontend/apps/reader/modules/readerfont.lua

```diff
@@ -405,6 +428,266 @@ function ReaderFont:onDecreaseFontSize(ges)
     return true
 end
 
+local font_family_info_text = _([[
+In HTML/CSS based documents like EPUBs, stylesheets can specify to use fonts by family name instead of a specific font name.
+Except for monospace and math, KOReader uses your default font for any family name.
+You can associate a specific font to each family name if you care about the distinction.
+Long-press on a font name will make the association global (★), so it applies on all your books. You should set a global association if you want to notice a book use font families.
```

Comment:
nit: applies `to`.

The second sentence on that line is a bit clunky, and redundant with the previous line, so I'd possibly drop it entirely?

Reply:
Also: `A long-press` or `Long-pressing` (I don't recall what we usually do, I tend to prefer the former).

Reply:
> The second sentence on that line is a bit clunky, and redundant with the previous line, so I'd possibly drop it entirely?

That was to insist on "better long-press than tap" as I mentionned at https://github.com/koreader/koreader/issues/7426#issuecomment-1259063939:

> When in the next book(s), family fonts will be unset, and no font family font used - so it may take time for people to notice they should long-press to get it global. Unlike most other features, for font-family, you don't notice the book uses font-family and you would need these unless they are already set and affect the rendering and show up.

So, rewording of the clunky part welcome - unless you still really feel it's not needed.

Reply:
`If you want to ensure that distinction is visible across your whole library, this is the preferred approach.`

Reply:
Your first part sounds even more redundant :)
```
global (★), so it applies on all your books
If you want to ensure that distinction is visible across your whole library
```
I could go with just adding "this is the preferred approach" - and let the users (who read help_text, so they must already be clever :) make you why it is :)

(I'd still like a hint that "hey dummy, if you don't long-press, you won't notice there are some families and you need to tap to see them" :)

Reply:
> I could go with just adding "this is the preferred approach" - and let the users (who read help_text, so they must already be clever :) make you why it is :)

Yup!